coalib/testing/LocalBearTestHelper.py

```diff
@@ -275,7 +295,9 @@ def check_line_result_count(self,
                                 force_linebreaks=True,
                                 create_tempfile=True,
                                 tempfile_kwargs={},
-                                settings={}):
+                                settings={},
+                                aspects=[],
```

Comment:
default to `None`, not empty list.

make adjustments anywhere else that breaks when it encounters `None`.